reference/pipeline/stages/index.md

```diff
@@ -169,6 +169,11 @@ Deploy a Kubernetes manifest YAML/JSON file.
 ### Find Artifacts From Resource
 Find artifacts from a Kubernetes resource.
 
+### Patch (Manifest)
+Update an already existing Kubernetes resource in place using the [Kubernetes patch operation](https://kubernetes.io/docs/tasks/run-application/update-api-object-kubectl-patch/). Spinnaker can update the resource without knowing the entire resource manifest i.e. you have to specify only the portion of the manifest you want to update. For instance, the patch stage can be used to add a label or update a sidecar container image for a set of resources. It can also be used to implement a red-black or "rainbow" deployment strategy for Kubernetes by first deploying a new ReplicaSet and then patching the fronting service's selectors to point to the new ReplicaSet.
```

Comment:
s/entire resource manifest i.e. you have to/entire resource (that is, you have to/

Plus a closing ) of course